.werft/util/gcloud.ts

```diff
@@ -33,3 +35,28 @@ export async function deleteExternalIp(phase: string, name: string, region = "eu
 function getExternalIp(name: string, region = "europe-west1") {
     return exec(`gcloud compute addresses describe ${name} --region ${region}| grep 'address:' | cut -c 10-`, { silent: true }).trim();
 }
+
+export function createDNSRecord(domain: string, dnsZone: string, IP: string, slice: string) {
```

Comment:
I'd change this to be an `async` function and then explicitly set the return type. So it would be the following:

```
export async function createDNSRecord(domain: string, dnsZone: string, IP: string, slice: string): Promise<void> {
```